infra/base-images/base-builder-fuzzbench/fuzzbench_run_fuzzer

```diff
@@ -29,6 +29,7 @@ export FUZZER=$FUZZING_ENGINE
 export MAX_TOTAL_TIME=120
 export SNAPSHOT_PERIOD=30
 export TRIAL_ID=1
+export FORCE_LOCAL=1
```

Comment:
Where does this come from? FuzzBench? If so can we document all of these in the comment (ideally link to FuzzBench documentation if available).

Reply:
Yep from FuzzBench. This isn't really an external API so there's no documentation.